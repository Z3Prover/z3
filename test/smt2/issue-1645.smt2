; Copyright (c) Levent Erkok
; https://github.com/Z3Prover/z3/issues/1645
(declare-fun x () Int)
(declare-fun y () Int)
(declare-fun z () Int)
(assert (< x z))
(assert (< y z))
(assert (< z 5))
(assert (distinct x y))
(maximize x)
(maximize y)
(check-sat)
(get-objectives)
(get-value (x))
(get-value (y))
(get-value (z))
