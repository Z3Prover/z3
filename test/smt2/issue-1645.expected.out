sat
(objectives
 (x 3)
 (y 2)
)
((x 3))
((y 2))
((z 4))
